tests/conftest.py

```diff
@@ -59,6 +61,12 @@ def repo(hermetic_seal):
         yield repo
 
 
+@pytest.fixture
+def short_tmpdir():
+    with tempfile.TemporaryDirectory() as tdir:
+        yield py.path.local(tdir)
```

Comment:
`py.path` is deprecated. Should this just use `from pathlib import Path` which is used elsewhere in this project?

Reply:
Thanks, I didn't know! I remember I was just glad to get the macOS tests working somehow..